.kitchen.yml

```diff
@@ -9,13 +9,68 @@ platforms:
   - name: ubuntu-14.04
     run_list:
     - "recipe[apt::default]"
+  - name: centos-7.1-vbox
```

Comment:
Understood. But really, I feel this should be a dependency of the Rundeck cookbook itself instead of having to plug that in at a higher level (like in Kitchen or in a Chef Role)